src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="big-serach" data-qa="big">
+      <label class="search" for="src-b">
+        <input
+        type="search"
+        name="search"
+        class="src src-big"
+        placeholder="Try &#34;Los Angeles&#34;"
+        id="src-b"
+        >
```

Comment:
First - remove autocomplete. It breaks input styles:
![image](https://user-images.githubusercontent.com/60885495/112498467-38efa000-8d8f-11eb-9174-398b72b82c3b.png)

Second - fix indentations:
```suggestion
        <input
          type="search"
          name="search"
          class="src src-big"
          placeholder="Try &#34;Los Angeles&#34;"
          id="src-b"
        >
```